pkg/handler/test/helper.go

```diff
@@ -142,7 +142,7 @@ const (
 	// RequiredEmailDomain required domain for predefined credentials
 	RequiredEmailDomain = "acme.com"
 	// DefaultKubernetesVersion kubernetes version
-	DefaultKubernetesVersion = "1.22.2"
+	DefaultKubernetesVersion = "1.22.4"
```

Comment:
pro tip: i would usually only change the versions in our tests when adding/removing a minor version. updating all tests just because we added a patch release leads to a needlessly large PR and now you need approval from the UI/API team as well 😁

Reply:
Yeah I get that, I just think I'll sleep better at night knowing that the new versions passed all of our tests. Kubernetes is super solid in their patch releases not breaking anything, but I'd still like the changes to be validated in our e2e 😄